cli/integration/tests/waiter/test_cli.py

```diff
@@ -497,3 +497,18 @@ def test_ping_custom_health_check_endpoint(self):
             self.assertEqual(1, len(util.services_for_token(self.waiter_url, token_name)))
         finally:
             util.delete_token(self.waiter_url, token_name)
+
+    def test_ping_timeout(self):
+        token_name = self.token_name()
+        util.post_token(self.waiter_url, token_name, util.minimal_service_description())
+        try:
+            cp = cli.ping(self.waiter_url, token_name, ping_flags='--timeout 50')
+            self.assertEqual(0, cp.returncode, cp.stderr)
+            self.assertIn('Successfully pinged', cli.stdout(cp))
+            util.kill_services_using_token(self.waiter_url, token_name)
+            cp = cli.ping(self.waiter_url, token_name, ping_flags='--timeout 1')
```

Comment:
nitpick: how do you feel about using kitchen startup delay and a slightly larger timeout, say 10s, for the failure scenario.

Reply:
Done